### PR TITLE
API-3768: practitioner openapi metadata

### PR DESCRIPTION
**Pull request by jjabando1**

### CODE REVIEW

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/metadata/R4MetadataController.java [hunk 1]
```diff
@@ -182,6 +182,11 @@ CapabilityStatement read() {
                 .profileUrl("http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient")
                 .search(Set.of(SearchParam.ID, SearchParam.IDENTIFIER))
                 .build(),
+            support("Practitioner")
+                .profileUrl(
+                    "http://hl7.org/fhir/us/core/STU3.1.1/StructureDefinition-us-core-practitioner.html")
```
Comment (eclendenning-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/metadata/R4MetadataController.java:187]: Wrong profile. http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner

File: data-query/src/test/resources/patient-r4-capability.json [hunk 2]
```diff
@@ -471,6 +471,35 @@
             }
           ]
         },
+        {
+          "type" : "Practitioner",
+          "profile" : "http://hl7.org/fhir/us/core/STU3.1.1/StructureDefinition-us-core-practitioner.html",
```
Comment (eclendenning-va) [inline data-query/src/test/resources/patient-r4-capability.json:476]: Here too, http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner

**Review by eclendenning-va — CHANGES_REQUESTED** [review]

File: data-query/src/main/java/gov/va/api/health/dataquery/service/api/Dstu2DataQueryService.java [hunk 3]
```diff
@@ -98,6 +99,9 @@
                           name = "patient/Observation.read",
                           description = "read observations"),
                       @OAuthScope(name = "patient/Patient.read", description = "read patient"),
+                      @OAuthScope(
+                          name = "patient/Practitioner.read",
```
Comment (jhulbert-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/api/Dstu2DataQueryService.java:103]: You need to add this scope towards the top of this file as well.

File: data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DataQueryService.java [hunk 4]
```diff
@@ -97,6 +98,9 @@
                           name = "patient/Observation.read",
                           description = "read observations"),
                       @OAuthScope(name = "patient/Patient.read", description = "read patient"),
+                      @OAuthScope(
+                          name = "patient/Practitioner.read",
```
Comment (jhulbert-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/api/R4DataQueryService.java:102]: Same in this file.

File: data-query/src/main/java/gov/va/api/health/dataquery/service/controller/metadata/R4MetadataController.java [hunk 5]
```diff
@@ -182,6 +182,10 @@ CapabilityStatement read() {
                 .profileUrl("http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient")
                 .search(Set.of(SearchParam.ID, SearchParam.IDENTIFIER))
                 .build(),
+            support("Practitioner")
+                .profileUrl("http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner")
+                .search(Set.of(SearchParam.ID, SearchParam.IDENTIFIER))
```
Comment (jhulbert-va) [inline data-query/src/main/java/gov/va/api/health/dataquery/service/controller/metadata/R4MetadataController.java:187]: R4 is id only.

**Review by jhulbert-va — APPROVED** [review]
Looks good.

**Pull request merged**